graph/step.go

```diff
@@ -257,8 +257,11 @@ func (s *Step) GetCmdWithCacheFlags(taskName string) (string, error) {
 	//  trim out any digests and tags
 	named = reference.TrimNamed(named)
 	domain, _ := reference.SplitHostname(named)
+	// todo: (sam) discuss if domain is empty, do we automatically attach $Registry?
+	// that might break behavior where customers use this image later on without $Registry
 	if domain == "" {
-		return result, errors.New("domain for the registry is required to push build cache")
+		result = fmt.Sprintf("--load %s", s.Build)
```

Comment:
if the build step has multiple tags like -t image1 -t image2, will the code load all of them?